package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "oav",
-  "version": "0.7.12",
+  "version": "0.7.13",
```

Comment:
per our conversation, this is happening, to see it being published via the pipeline.